Competitive_Programming/C++/Bit Manipulation/README.md

```diff
@@ -0,0 +1 @@
+`Bit Manipulation`
```

Comment:
Do not edit the README.md. We will be doing it later in a systematic order to maintain the consistency of the repository.